pkgs/by-name/sk/skyscraper/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  qt5,
+  p7zip,
+  python3,
+
+  # Whether to compile with XDG support
+  # (See: https://gemba.github.io/skyscraper/XDG/)
+  enableXdg ? false,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "skyscraper";
+  version = "3.14.0";
+
+  src = fetchFromGitHub {
+    owner = "Gemba";
+    repo = "skyscraper";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-UgPo9qEsYYYl2RgdPtPmToAruPKuvJ3FSgVrXa94kdA=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    qt5.qmake
+  ];
+
+  buildInputs = [ python3 ];
+
+  postPatch = lib.optionalString enableXdg ''
+    substituteInPlace skyscraper.pro --replace-fail "#DEFINES+=XDG" "DEFINES+=XDG"
+  '';
+
+  preFixup = ''
+    qtWrapperArgs+=(--prefix PATH : ${lib.makeBinPath [ p7zip ]})
+  '';
+
+  env.PREFIX = placeholder "out";
+
+  meta = {
+    description = "Powerful and versatile game data scraper written in Qt and C++";
+    homepage = "https://gemba.github.io/skyscraper/";
+    downloadPage = "https://github.com/Gemba/skyscraper/releases/tag/${finalAttrs.version}";
+    license = lib.licenses.gpl3Only;
```

```suggestion
    license = lib.licenses.gpl3Plus;
```
Several files state (or later).

---

Done.